python-checks/src/test/resources/checks/numpyWhereOneCondition.py

```diff
@@ -0,0 +1,47 @@
+def import_1():
+    import numpy as np
+    arr = np.array([1, 2, 3, 4])
+
+    empty = np.where()
+    args = (arr > 1, arr + 1, arr)
+    result = np.where(*args)
```

Comment:
Maybe we can add a test with the tuple `(arr > 1)` to see if the issue is still raised. I think that would be quite nice as it is a less obvious mistake.